core/field_number.js

```diff
@@ -270,29 +296,25 @@ class FieldNumberPicker extends Blockly.FieldNumber {
     e.preventDefault();
   }
 
-  doClassValidation_(
-    newValue
-  ) {
-    if (newValue === '') return newValue;
-    return super.doClassValidation_(newValue);
+  doClassValidation_(newValue) {
+    // We validate at the point of input (see onHtmlInputKeyDown_), so
```

Comment:
Can you add info here, in the other method, or in the PR description about why we decided to do it this way (I.e. the validation flashing) so the Future Generations can understand?

Reply:
I think this is resolved by not overriding/reenabling the default validation and thus removing this comment?